apps/vs-code-designer/src/app/utils/codeless/startDesignTimeApi.ts

```diff
@@ -178,3 +194,34 @@ export function stopDesignTimeApi(): void {
   }
   ext.workflowDesignChildProcess = undefined;
 }
+
+export async function promptStartDesignTimeOption(context: IActionContext) {
+  if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 0) {
+    const workspace = await getWorkspaceFolder(context);
+    const projectPath = await tryGetFunctionProjectRoot(context, workspace);
+    const autoStartDesignTimeKey = 'autoStartDesignTime';
```

Comment:
There is a space for settings keys in the constants.ts. Could you please put them over there?